models/oneinch/oneinch_filter_tokens.sql

```diff
@@ -0,0 +1,43 @@
+{{ config(materialized='view', alias='filter_tokens') }}
+
+-- last updated 2023-04-17
+with broken_prices as (
```

Comment:
interesting! for your awareness, we've slowly [been building an exclusion list](https://github.com/duneanalytics/spellbook/pull/3213/files#diff-c19899e75831b0223fbd3204e898a28798e8ca754edb3ca290000f0c4625c453R1620) in the ethereum prices spell, which includes overlap of some tokens here that we've seen issues with too. would you prefer to remove this view and add these addresses to each of those prices files? that way you can avoid this view and extra join to it everywhere?